### PR TITLE
[2.19.x] G-9823 add listener for rendering shapes downstream

### PR DESCRIPTION
**Pull request by andrewzimmer**
#### What does this PR do?
adds listener for rendering shapes downstream

#### How should this be tested?
- build / install
- verify no regression drawing shapes on map

#### What are the relevant tickets?
Fixes: #____

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Review by brianfelix — APPROVED** [review]

**Review by pklinef — APPROVED** [review]

File: ui/packages/catalog-ui-search/src/main/webapp/react-component/filter/filter-input/filter-location-input.js [hunk 1]
```diff
@@ -79,6 +79,11 @@ class LocationInput extends React.Component {
     this.props.listenTo(this.locationModel, 'change:mode', () => {
       this.clearLocation()
     })
+    this.props.listenTo(this.locationModel, 'change:polygon', () => {
+      if (this.locationModel.get('mode') !== 'poly') {
+        wreqr.vent.trigger('search:polydisplay', this.locationModel)
```
Comment (mojogitoverhere) [inline ui/packages/catalog-ui-search/src/main/webapp/react-component/filter/filter-input/filter-location-input.js:84]: I think I'm just missing something but this seems odd. We only want to trigger the `polydisplay` if the mode is _not_ `poly`?

**Review by rymach — APPROVED** [review]

File: ui/packages/catalog-ui-search/src/main/webapp/react-component/filter/filter-input/filter-location-input.js (hunk 1 above)
Comment (mojogitoverhere) [reply ui/packages/catalog-ui-search/src/main/webapp/react-component/filter/filter-input/filter-location-input.js:84]: Resolved offline. This fixes the aoi/target rendering issue, which has the modes `aoi` / `target`.

**Review by mojogitoverhere — APPROVED** [review]

**Comment by andrewzimmer** [comment]
build now

**Comment by mojogitoverhere** [comment]
hero in progress

**Comment by mojogitoverhere** [comment]
## 🎉 Hero build was successful! 🎉 ##

Verified fix in downstream project

**Pull request merged**